packages/astro/src/core/render/dev/index.ts

```diff
@@ -133,7 +133,11 @@ async function getScriptsAndStyles({ env, filePath }: GetScriptsAndStylesParams)
 		});
 		// But we still want to inject the styles to avoid FOUC
 		styles.add({
-			props: {},
+			props: {
+				type: 'text/css',
```

Comment:
is this because vite needs the attributes to match? if not, `type` is not need on script or style, `<style>` is always text/css.

Reply:
No, `data-astro-dev-id` is the only _necessary_ change here. I just added this for consistency with the styles Vite injects.